faktorio-api/src/routers/invoices/invoiceRouter.ts

```diff
@@ -205,7 +205,10 @@ export const invoiceRouter = trpcContext.router({
         where: and(...conditions),
         limit: input.limit ?? undefined,
         offset: input.offset ?? undefined,
-        orderBy: desc(invoicesTb.taxable_fulfillment_due) // Order by taxable date
+        orderBy: [
+          desc(invoicesTb.taxable_fulfillment_due),
+          desc(invoicesTb.created_at)
```

Comment:
happened to me, tha twhen a invoice was updated, the order was changed. 
The problem is that taxable_fulfillment_due is last day of hte previous month. Not that flexible...

Reply:
i am not sure if the array here is corredct but it seems to bre working